openedx/core/djangoapps/appsembler/sites/utils.py

```diff
@@ -481,7 +464,6 @@ def bootstrap_site(site, org_data=None, username=None):
         organization_data = org_api.add_organization({
             'name': organization_slug,
             'short_name': organization_slug,
```

Comment:
This line is harmless, even with the upstream edx-organization.

```suggestion
            'short_name': organization_slug,
            'edx_uuid': org_data.get('edx_uuid')  # TODO: RED-2845 Remove this line when AMC is migrated
```